zul/src/archive/web/js/zul/layout/Borderlayout.js

```diff
@@ -299,6 +306,9 @@ zul.layout.Borderlayout = zk.$extends(zul.Widget, {
 		el._lastSize = {width: w, height: h};
 		return false;
 	},
+	_adjustVerticalTitle: function (region) {
+		jq(region.$n('title')).width(jq(region.$n('colled')).height() * 0.8);
```

Comment:
Hard-coded 0.8 is not a good idea.
Can this multiplier be extracted to somewhere?

Or let the width as large as possible (by jq(region.$n('colled')).height() - button).